operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -47,6 +47,45 @@ It may fail to identify installed components or vulnerabilities if the final ima
 
 include::modules/scanning-images.adoc[leveloffset=+1]
 
+include::modules/accessing-delegated-image-scanning.adoc[leveloffset=+1]
+
+include::modules/enhancing-image-scanning-by-accessing-delegated-image-scanning.adoc[leveloffset=+2]
+
+include::modules/configuring-delegated-image-scanning.adoc[leveloffset=+2]
+
+[id="installing-and-configuring-scanning-on-secured-clusters_{context}"]
+=== Installing and configuring scanning on secured clusters
+
+[id="using-the-operator_{context}"]
+==== Using the Operator
+
+{product-title-short} Operator installs a Scanner-slim version on each secured cluster to scan images in the {ocp} integrated registry and optionally other registries.
+
+For more information, see xref:../installing/installing_ocp/install-secured-cluster-ocp.adoc#installing-sc-operator[Installing {product-title-short} on secured clusters by using the Operator].
+
+
+[id="using-helm_{context}"]
+==== Using Helm
+
+Secured Cluster Services Helm chart (`secured-cluster-services`) installs a Scanner-slim version on each secured cluster.
+In Kubernetes, the secured cluster services include Scanner-slim as an optional component.
+On {ocp}, however, {product-title-short} installs a Scanner-slim version on each secured cluster to scan images in the {ocp} integrated registry and optionally other registries.
+
+* For {ocp} installations, see xref:../installing/installing_ocp/install-secured-cluster-ocp.adoc#installing-secured-cluster-services-quickly_install-secured-cluster-ocp[Installing the secured-cluster-services Helm chart without customization].
+
+* For non-{ocp} installations footnote:disclaimer[Platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE), and Microsoft Azure Kubernetes Service (Microsoft AKS).], see xref:../installing/installing_other/install-secured-cluster-other.adoc#installing-secured-cluster-services-quickly_install-secured-cluster-other[Installing the secured-cluster-services Helm chart without customization].
+
+
+[id="verifying-after-installation_{context}"]
+==== Verifying after installation
+
+.Procedure
+
+* Verify that the status of the secured cluster indicates that Scanner is present and healthy:
+** In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters*.
+** In the *Clusters* view, select a cluster to view its details.
+** In the *Health Status* card, ensure that *Scanner* is present and is marked as *Healthy.*
```

Comment:
Although it's a good call to make the main step unordered because it's a single step, these nested sub-steps [should be ordered because the sequence of actions is important](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#ordered-lists).